src/reportingApi.ts

```diff
@@ -0,0 +1,30 @@
+// eslint-disable-next-line import/named
+import { Task } from 'vitest';
+import * as Models from './models';
+import { isRPTaskMeta } from './utils';
+
+const injectRPTaskMeta = (task: Task) => {
+  if (isRPTaskMeta(task.meta)) {
+    return;
+  }
+
+  (task.meta as Models.RPTaskMeta) = {
+    ...task.meta,
+    test: {
```

Comment:
I suppose such general name as `test` can lead to a name collision with custom user's `meta`.
Let's use more rp-specific name, f.e. `rpMeta` or smth like that.